processor/k8sprocessor/config.go

```diff
@@ -173,3 +175,15 @@ type FieldFilterConfig struct {
 	//   equals, not-equals, exists, does-not-exist.
 	Op string `mapstructure:"op"`
 }
+
+// PodAssociationConfig allows specifying rules for associating resources
+// with pod metadata
+type PodAssociationConfig struct {
+	// From represents the source of the association.
+	// Allowed values are connection and labels
+	From string `mapstructure:"from"`
+
+	// Name represents the name of the association.
+	// e.g. ip, pod_uid
```

Comment:
What are the possible values? Is "ip" and "pod_uid" the only possible values?

Reply:
No, there are few more